collection-scripts/targeted_logs

```diff
@@ -99,6 +100,22 @@ for nsdv in ${target_dvs[@]}; do
   fi
 done
 
+# Collect CDI/importer pod logs for imported PVCs
```

Comment:
can we filter the PVCs that are based on DVs included in target_dvs so we'll operate only on PVCs that we don't expect to have importer pods for?

Reply:
We pass on the VM and look for the volumes. It can be either DV or PVC. We can look into the PVC and understand it came from CDI (and therefore filter it). But it will miss the purpose of getting the importer pods.

The only option to change it is when looking into the vm details if we find a PVC that came from CDI to append it into the DV resources instead of the PVC resources - it feels a bit off to me.

Reply:
I will query the PVC and see it made by CDI. e.g having the label: `app: containerized-data-importer`

Reply:
filtering out those PVCs by looking at their labels sound good, but please try to avoid adding another query - it would be better to use the information that was dumped in dump_resource

Reply:
we don't have the related information we need

Reply:
@liranr23 can you be more specific? you mean that we don't get the PVC with its labels by running `/usr/bin/oc get ${resource} -n ${ocproject} -o yaml ${ocobject}` ?

Reply:
in the dump_resource we dump the pvc name. yes, the command you wrote is another query that will get us what we need

Reply:
ack, yeah so I referred to [that call in dump_resource](https://github.com/kubev2v/forklift-must-gather/blob/main/collection-scripts/targeted_crs#L96)